contracts/constants.cairo

```diff
@@ -22,7 +22,7 @@ const ACCOUNT_BALANCE_UPPER_BOUND = 2 ** 64 * Math64x61.FRACT_PART;
 // The minimum and maximum volatility
 const VOLATILITY_LOWER_BOUND = 1;
 const VOLATILITY_UPPER_BOUND = 2 ** 64 * Math64x61.FRACT_PART;
-
+const MAX_OPTION_SIZE_PERCENT_OF_VOLADJSPD = 20;
```

Comment:
put commnet above describing what this is

Reply:
also, wouldn't it be better to make it adjustable? so we can effectively turn it off in tests??? otherwise we have to rewrite bunch of them

Reply:
hmmm... I guess you are right... could be done similarly like `volatility_adjustment_speed: Math64x61_` in `add_lptoken`